executorlib/base/executor.py

```
@@ -0,0 +1,148 @@
+        Returns:
+            Optional[dict]: Information about the executor.
+        """
+        if self._process is not None and isinstance(self._process, list):
+            meta_data_dict = self._process[0]._kwargs.copy()
+            if "future_queue" in meta_data_dict.keys():
+                del meta_data_dict["future_queue"]
+            meta_data_dict["max_workers"] = len(self._process)
+            return meta_data_dict
+        elif self._process is not None:
+            meta_data_dict = self._process._kwargs.copy()
+            if "future_queue" in meta_data_dict.keys():
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)